PWGHF/HFC/Tasks/taskCorrelationDplusHadrons.cxx

```diff
@@ -162,6 +162,7 @@ struct HfTaskCorrelationDplusHadrons {
     registry.add("hBdtScorePrompt", "D+ BDT prompt score", {HistType::kTH1F, {axisBdtScore}});
     registry.add("hBdtScoreBkg", "D+ BDT bkg score", {HistType::kTH1F, {axisBdtScore}});
     registry.add("hMassDplusVsPt", "D+ candidates massVsPt", {HistType::kTH2F, {{axisMassD}, {axisPtD}}});
+    registry.add("hMassDplusVsPtWoeff", "D+ candidates massVsPt without efficiency", {HistType::kTH2F, {{axisMassD}, {axisPtD}}});
```

Comment:
Can you modify the histogram name using the camel case convention and propagate it to the other occurrences in the class?
```suggestion
    registry.add("hMassDplusVsPtWoEff", "D+ candidates massVsPt without efficiency", {HistType::kTH2F, {{axisMassD}, {axisPtD}}});
```